deblur/sequence.py

```diff
@@ -0,0 +1,49 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2013, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from collections import defaultdict
+import re
+
+import numpy as np
+
+trans_dict = defaultdict(lambda: 5, A=0, C=1, G=2, T=3)
+trans_dict['-'] = 4
+
+
+class Sequence(object):
```

Comment:
@wasade the problem that I have is that I want to precompute some attributes of the Sequence, and right now I don't have an easy way to add that information to the sequence object.